titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/JooqConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.ext.jooqflyway.jobactivity;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+@Configuration(prefix = "titus.ext.jooqflyway")
+public interface JooqConfiguration {
```

Comment:
Should this be named something more specific, like `JooqJobActivityConfiguration`? To avoid ambiguity with the existing `JooqConfiguration`?

Reply:
its all scoped to jobactivity